torch/overrides.py

```diff
@@ -38,6 +37,7 @@ def get_ignored_functions() -> Set[Callable]:
     be overridden with __torch_function__. Mostly this is because none of the
     arguments of these functions are tensors or tensor-likes.
     """
+    import torch
```

Comment:
Why do you have to move this?
Is it because it's imported in autograd now? Doesn't the python import logic handle this already since you don't use anything in `torch` in this file outside of the functions?

Reply:
You're right, I don't. I was being overly cautious.